understanding/21/reflow.html

```diff
@@ -16,64 +16,286 @@ <h2>In brief</h2>
+
+          <figure>
+            <img src="img/..." alt="TODO">
+            <figcaption>
+              <p>A table is preceded by a heading, and followed by a pagination component. The table is excepted from Reflow, and is rendered within a scrollable container. The heading and pagination are not excepted, and thus adjust to fit within a zoomed in viewport.</p>
+            </figcaption>
+          </figure>
+
+          <h4>Interfaces with persistent toolbars</h4>
+
+          <p>Interfaces which provide toolbars to edit content need to show both the content and the toolbar in the viewport. Depending on the number of toolbar buttons, the toolbar may need to scroll in the direction of text, or might even need to remain fully visible and scroll along with the rich text content or canvas area that it provides features for editing.</p>
+
+          <figure>
+            <img src="img/..." alt="TODO">
+            <figcaption>
+              <p>TODO: example of a persistent toolbar interface</p>
+            </figcaption>
+          </figure>
+
+          <h4>Layout needed for usage and meaning</h4>
+
+          <p>Complex web application interfaces allow users to create content set to defined dimensions. For instance, presentations or documents based on physical printing or display sizes. These interfaces often allow for other related content to be created that is directly related content to the primary author created content, but not part of the author created content. For instance, presentation notes, editing comments/suggestions for a text document, dynamic messaging presented along with a message (chat) interface, etc. Each of these related sections of content often need to be presented in parallel to each other, and their understandability can diminish if their overall location were to change.</p>
+
+          <figure>
+            <img src="img/rich-document-with-comment.png" alt="online rich text document - on the left: the text document is displayed at dimensions representing the physical width and height of the page.  On the right, a comment referring to highlighted text in the text document.">
+            <figcaption>
+              <p>A rich text document is presented at the height and width dimensions which represent how the page would be formatted if physically printed. A piece of text is highlighted, indicating it has a comment associated with it. This comment (a section of content) is located to the side of the rich text document in its own column.  This comment does not need to maintain a specific height or width since it only exists as part of this interface, and thus this comment is expected to reflow so that it can fit within a 320 CSS pixel wide viewport, while the rich text document can maintain its necessary height and width.</p>
```

Comment:
```suggestion
              <p>A rich text document is presented at the height and width dimensions which represent how the page would be formatted if physically printed. A piece of text is highlighted, indicating it has a comment associated with it. This comment (a section of content) is located to the side of the rich text document in its own column.  This comment does not need to maintain a specific height or width since it only exists as part of this interface. Therefore, this comment is expected to reflow so that it can fit within a 320 CSS pixel wide viewport, while the rich text document can maintain its necessary height and width.</p>
```